tsconfig.json

```diff
@@ -7,8 +7,6 @@
     "declarationMap": true,
     "forceConsistentCasingInFileNames": true,
     "importHelpers": true,
-    "lib": ["esnext"],
```

Comment:
ts-node failed with 'esnext'. Why do we compile to esnext? I removed it because I don't think it's necessary but let me know.